src/Aspire.Dashboard/ConsoleLogs/AnsiParser.cs

```diff
@@ -21,6 +21,24 @@ public class AnsiParser
     private const int XtermForegroundSequenceCode = 38;
     private const int XtermBackgroundSequenceCode = 48;
 
+    public static string StripControlSequences(string text)
```

Comment:
This is very inefficent. Compare it with `ConvertToHtml` which:

* Jumps directly to the next escape char using `IndexOf`. `IndexOf` is very fast and it avoids inspecting every character multiple times
* Adds ranges of text. Avoids appending one char at a time to the StringBuilder

Additionally, if there are no control sequences (will be common) then there is no need to create a StringBuilder. The original string can be returned.

Reply:
Gotcha, made this more efficient now.

Reply:
Much better. I made a couple of small changes so that the StringBuilder can be lazy, and the builder is initialized at the existing strings length.